pkg/provider/aws/action/openshift-snc/constants.go

```diff
@@ -25,7 +25,8 @@ var (
 	outputKubeAdminPass  = "aosKubeAdminPasss"
 	outputDeveloperPass  = "aosDeveloperPass"
 
-	commandReadiness    = "while [ ! -f /tmp/.crc-cluster-ready ]; do sleep 5; done"
+	commandSSHReadiness = "echo 'SSH connectivity established'"
```

Comment:
May you can reuse 

https://github.com/redhat-developer/mapt/blob/4adadf0d788bea0d7cae2f58648e46a9784e8589/pkg/provider/util/command/command.go#L19

... or if it is more clear using this echo instead of ping... you can change it there and reuse it from here. I am fine with either decission.